src/SFA.DAS.Commitments.Database/AdhocScripts/Manual/CV-684-Update-Account-LevyStatus.sql

```diff
@@ -0,0 +1,11 @@
+﻿/*
+CV-684 - Initial set of Account levy status
+Usage instructions:
+Execute the below againsts the Accounts db.
+Copy-paste the resulting output (a set of update statements), and execute those against the Commitments database.
+*/
+
+select
+'update [Accounts] set [LevyStatus]=0 where [Id]=' + convert(varchar,Id)
+from [employer_account].[Account]
+where ApprenticeshipEmployerType = 0
```

Comment:
Since the default for LevyStatus in our DB is 0, shouldn't this be  

select
'update [Accounts] set [LevyStatus]=1 where [Id]=' + convert(varchar,Id)
from [employer_account].[Account]
where ApprenticeshipEmployerType = 1

Reply:
As per discussion in slack, the post deployment script will mark accounts as levy and then we will set them back again, so that before this script runs the system will continue to work for levy payers.